_includes/corridors/xof-mobile.md

```diff
@@ -90,6 +90,28 @@ tmoney
 
 {% include language-tabbar.html prefix="xof-mobile-providers" raw=data-raw %}
 
+All senders trying to create XOF::mobile Togo payouts need to have the `country` and `phone_number` fields present:
+- `"phone_number" => "+15555551234"`
+
+Please note that the field above is generally considered optional for senders for other payment corridors (WTR2 rules apply - kindly refer to the [sender WTR2 rules]({{ "/docs/transaction-flow/" | prepend: site.baseurl }}#wtr2) section for further details). If you wish to use an existing sender who's currently missing the `country` field you can provide it alongside the `id` or `external_id` field in the sender details. For example:
```

Comment:
The `country` is not tied to `WTR2` rules, it's required at all times so mentioning it alongside the `phone_number` seems misleading

Reply:
Thank @cesarecamurani for noting this. Let me run it by product. I was following the ticket details here https://azafinance.atlassian.net/browse/BTA-15793